app/lib/actions/katello/content_view_version/destroy.rb

```diff
@@ -4,15 +4,18 @@ module ContentViewVersion
       class Destroy < Actions::Base
         def plan(version, options = {})
           version.check_ready_to_destroy! unless options[:skip_environment_check]
+          destroy_env_content = !options.fetch(:skip_destroy_env_content, false)
+          repos = destroy_env_content ? version.repositories : version.archived_repositories
```

Comment:
should be `version.archived_repos` 

`archived_repositories` is not a method in -> https://github.com/Katello/katello/blob/master/app/models/katello/content_view_version.rb#L123